Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -1,19 +1,164 @@
-.. |author| replace:: *TDB*
+.. |author| replace:: *Yijung Kang*
 .. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
-.. |contributors| replace:: *none*
+.. |contributors| replace:: *Alysha Shugart, Karla Aubel*
 
 .. _AT-Shutdown-Shutdown:
 
 #######################
 Shutdown
 #######################
 
-Under construction.
+
+.. _Shutdown-the-Telescope-Overview:
+
+Overview
+========
+
+At the end of the night, the telescope, LATISS, CSCs compoents, and Dome should be in shutdown and/or standby status.
+The parking position is for the AuxTel is El = 80 deg, Az = 0.0 deg, rot = 0.0 deg. 
+For the dome, it is at az = 285.0 deg with closed dome slit shutter.
+All ATTCS and LATISS CSCs for AuxTel should be set to standby.  
+
```

Comment:
I would re frase it like: "At the end of the night  ATTCS and LATISS CSCs should be set to standby. Telescope&dome should be parked..."and then you the position of each...